spec/integration/integration.spec.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const path = require('path');
+const expect = require('chai').expect;
+const services = require('../helpers/services');
+
+const LONG_TIMEOUT = 30000;
+
+describe('LocalstackPlugin', () => {
+
+  it('should deploy a stack', () => {
+    let service = services.createService({});
+    services.deployService(service);
```

Comment:
There should be an assertion here to validate that it deployed correctly.

This is mainly to ensure that the intention of the plugin works correctly.  Agree that there is a bit of overlap with Localstack testing, but I see it as an acceptance test for the plugin.